services/nextjs-app/sst.config.ts

```diff
@@ -0,0 +1,20 @@
+import { SSTConfig } from "sst";
+import { NextjsSite } from "sst/constructs";
+
+export default {
+  config(_input) {
+    return {
+      name: "nextjs-app",
```

Comment:
can you change the name to web-app.

Also the name of the folder as well :)

Reply:
done :)